From 25138a275a4a39aef9748dadc3fdd4816a48fddb Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@windriver.com>
Date: Thu, 14 Oct 2010 16:38:00 -0400
Subject: [PATCH] powerpc/boot: remove -Os from BOOTFLAGS

Some variants of gcc 4.5 have issues with -Os and will
throw:

  cc1: error: Do not use -Os option if --enable-target-optspace is not set.

-Os is not required to build a valid boot environment,
so the easiest solution is to simply remove the boot
flag.

Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>

diff --git a/arch/powerpc/boot/Makefile b/arch/powerpc/boot/Makefile
index 90e9d9548660..af10ceec0cab 100644
--- a/arch/powerpc/boot/Makefile
+++ b/arch/powerpc/boot/Makefile
@@ -20,7 +20,7 @@
 all: $(obj)/zImage
 
 BOOTCFLAGS    := -Wall -Wundef -Wstrict-prototypes -Wno-trigraphs \
-		 -fno-strict-aliasing -Os -msoft-float -pipe \
+		 -fno-strict-aliasing -msoft-float -pipe \
 		 -fomit-frame-pointer -fno-builtin -fPIC -nostdinc \
 		 -isystem $(shell $(CROSS32CC) -print-file-name=include) \
 		 -mbig-endian
-- 
2.1.2

